jobs/openshift/cluster/integreatly/test/Jenkinsfile

```diff
@@ -0,0 +1,124 @@
+  }
+
+  stages {
+    stage('Verify Parameters') {
+      steps {
+        script {
+          testOptions.awsAccountName = 'fheng.AWS'
+          testOptions.awsRegion = 'eu-west-2'
+          testOptions.clusterName = params.installerGitBranch
+          testOptions.domainName = 'skunkhenry.com'
+          testOptions.gitUrl = params.installerGitUrl
+          testOptions.gitBranch = params.installerGitBranch
+          testOptions.openshiftMasterUrl = "https://${params.installerGitBranch}.${testOptions.domainName}"
+          verifyParameters(testOptions)
+        }
+      }
+    }
+
+    stage('Cluster Create') {
+      steps {
+        echo "[INFO] Creating ${testOptions.clusterName} cluster"
+        script {
+          def jobName = 'openshift-cluster-create'
+          def jobParams = [
+            [$class: 'StringParameterValue', name: 'clusterName', value: testOptions.clusterName],
+            [$class: 'StringParameterValue', name: 'awsRegion', value: testOptions.awsRegion],
+            [$class: 'StringParameterValue', name: 'awsAccountName', value: testOptions.awsAccountName],
+            [$class: 'BooleanParameterValue', name: 'dryRun', value: params.dryRun]
+          ]
+          build job: jobName, parameters: jobParams
+        }
+      }
+    }
+
+    stage('Install Integreatly') {
+      steps {
+        echo "[INFO] Installing Integreatly in ${testOptions.clusterName} cluster"
+        script {
+          jobName = 'openshift-cluster-integreatly-install'
+          jobParams = [
+            [$class: 'StringParameterValue', name: 'clusterName', value: testOptions.clusterName],
+            [$class: 'StringParameterValue', name: 'openshiftMasterUrl', value: testOptions.openshiftMasterUrl],
+            [$class: 'StringParameterValue', name: 'installerGitUrl', value: testOptions.gitUrl],
+            [$class: 'StringParameterValue', name: 'installerGitBranch', value: testOptions.gitBranch],
+            [$class: 'BooleanParameterValue', name: 'dryRun', value: params.dryRun]
+          ]
+          build job: jobName, parameters: jobParams
+        }
+      }
+    }
+
+    stage('Test Integreatly') {
+      steps {
+        echo "[INFO] Running Tests on Integreatly"
+      }
+    }
+
+    stage('Uninstall Integreatly') {
+      steps {
+        echo "[INFO] Uninstalling Integreatly from ${testOptions.clusterName}"
+        script {
+          jobName = 'openshift-cluster-integreatly-uninstall'
+          jobParams = [
+            [$class: 'StringParameterValue', name: 'clusterName', value: testOptions.clusterName],
+            [$class: 'StringParameterValue', name: 'openshiftMasterUrl', value: testOptions.openshiftMasterUrl],
+            [$class: 'StringParameterValue', name: 'uninstallerGitUrl', value: testOptions.gitUrl],
+            [$class: 'StringParameterValue', name: 'uninstallerGitBranch', value: testOptions.gitBranch],
```

Comment:
If we rename these here https://github.com/integr8ly/ci-cd/pull/117#discussion_r274423390, they will need changed here also.

Reply:
Will address the param naming in a follow up PR.